pkgs/development/tools/gomacro/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gomacro";
+  version = "2.7";
+
+  src = fetchFromGitHub {
+    owner = "cosmos72";
+    repo = "gomacro";
+    rev = "b4c3ab9b218fd12f22759354f4f3e37635828d1f";
```

If the mentioned change from above is used. 

```suggestion
    inherit rev;
```